examples/lines_bars_and_markers/markevery_demo.py

```diff
@@ -46,47 +46,42 @@
 ###############################################################################
 # Plot each markevery case for linear x and y scales
 
-fig1 = plt.figure(num=1, figsize=figsize)
-ax = []
+fig1, axs = plt.subplots(rows, cols, figsize=figsize, constrained_layout=True)
+axs = axs.flat
+axs[-1].remove()  # not used...
 for i, case in enumerate(cases):
-    row = (i // cols)
-    col = i % cols
-    ax.append(fig1.add_subplot(gs[row, col]))
-    ax[-1].set_title('markevery=%s' % str(case))
-    ax[-1].plot(x, y, 'o', ls='-', ms=4, markevery=case)
+    ax = axs[i]
```

Comment:
`for ax, case in zip(axs, cases):` (don't think `i` is used for anything here now.)